validation-test/compiler_crashers_2/0083-rdar31163470-1.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: not --crash %target-swift-frontend -primary-file %s -emit-ir
+// REQUIRES: asserts
+
+struct First<T> {}
+struct Second<T> {}
+
+struct Node<T> {
+  func create<U>() where T == First<U> { }
```

Comment:
I didn't intentionally make that change. It should have only dropped requirements *from the outer scope*.